guides/v2.3/graphql/reference/wishlist.md

```diff
@@ -12,14 +12,14 @@ Use Wishlist queries to retrieve information about a customer's wish list.
 
 Attribute | Data type | Description
 --- | --- | ---
-`items` | [Wishlistitem](#wishlistitem) | An array of items in the customer's wish list
+`items` | [WishlistItem](#WishlistItem) | An array of items in the customer's wish list
 `items_count` | Int | The number of items in the wish list
 `name` | String | When multiple wish lists are enabled, the name the customer assigns to the wish list
 `sharing_code` | String | An encrypted code that Magento uses to link to the wish list
 `updated_at` | String | The time of the last modification to the wish list
 {:style="table-layout:auto;"}
 
-### Wish list item attributes {#wishlistitem}
+### Wish list item attributes {#WishlistItem}
```

Comment:
This anchor link should not be changed. Please remove this part of your edit.

Reply:
Done